client/galaxy/scripts/mvc/workflow/workflow-forms.js

```diff
@@ -236,7 +236,7 @@ define( [ 'utils/utils', 'mvc/form/form-view', 'mvc/tool/tool-form-base' ], func
                             for ( var p_id in input.payload ) {
                                 var p = input.payload[ p_id ];
                                 input.payload[ input.name + '__' + p_id ] = p;
-                                delete p;
+                                delete input.payload[ p_id ];
```

Comment:
@guerler Actually, wasn't from a bad merge, I think I just fat fingered exiting my editor prior to commit in 3374d21904f0293b7b16735210bbdbf842504be6.